addons/udes_stock_routing/models/stock_move.py

```diff
@@ -165,7 +165,6 @@ def _split_pick_hook(self, applicable_rules):
         for move in self:
             move_qty = move.product_uom_qty
             # Post assign actions could change the picking of assigned moves, so use move instead.
```

Comment:
Nit: Is this comment needed here anymore?

Reply:
I'll review and remove later :+1: good spot